src/main/kotlin/no/nav/k9/los/nyoppgavestyring/mottak/oppgave/OppgaveV3.kt

```diff
@@ -69,6 +69,10 @@ class OppgaveV3(
             }
             return oppgavefelter
         }
+
+        fun gjelderFRISINN( oppgave : OppgaveV3) : Boolean {
```

Comment:
Ikke helt ideélt å ha frisinn (eller ytelsespesifikke ting) på den generiske OppgaveV3, men alternativet blir fort en del mer kode. 
F.eks at vi har et felt på oppgaven som settes av adapterne, som avgjør om oppgaven skal skjules i sin helhet i los. Et flagg inn i `sjekkDuplikatOgProsesser` kunne også vært en mulighet, men tror feltdefinisjoner er det som er mest riktig med tanke på hvordan resten av los virker.

Reply:
Jeg har forsøkt et kompromiss nå ved å flytte koden fra OppgaveV3 til FrisinnUtil

Jeg tror dette er den beste løsningen inntil alle hendelsene for FRISINN slettes fra LOS, siden vi da er helt sikre på at de ikke dukker opp i køene, ikke påvirker ytelse for køene, og er enkelt å forstå (tror jeg)

Reply:
Ja, enig, det er et greit kompromiss. Det hadde nok blitt unødvendig mye kode med den løsningen jeg foreslo